app/views/patients/show.html.erb

```diff
@@ -6,9 +6,18 @@
 
 <%= link_to "Edit Patient", edit_district_facility_patient_path(@district, @facility, @patient), class: "btn btn-secondary float-right" %>
 
-<h1>
-  Treatment Number <%= @patient.formatted_treatment_number %>
-</h1>
+<div class="row">
+  <div class="col-md-9">
+    <h1>Treatment Number <%= @patient.formatted_treatment_number %></h1>
+  </div>
+  <div class="col-md-2">
```

Comment:
Once we merge user policy we should show these badges only to admins.

Reply:
Shall I add a TODO here?